README.md

```diff
@@ -165,13 +165,13 @@ The plugin supports the following queries:
 
 ```scala
 import akka.actor.ActorSystem
-import akka.stream.{Materializer, ActorMaterializer}
+import akka.stream.Materializer
 import akka.stream.scaladsl.Source
 import akka.persistence.query.PersistenceQuery
 import akka.persistence.jdbc.query.scaladsl.JdbcReadJournal
 
 implicit val system: ActorSystem = ActorSystem()
-implicit val mat: Materializer = ActorMaterializer()(system)
+implicit val mat: Materializer = Materializer.matFromSystem(system)
```

Comment:
Preferrably even skip the implicit mat at all, it should be available just by having the system implicit.

Reply:
sure, I will revisit all this.